src/tests/presale/test_checkout.py

```diff
@@ -151,6 +151,28 @@ def _enable_country_specific_taxing(self):
         self._set_session('invoice_address', ia.pk)
         return ia
 
+    def test_pci_page(self):
+        with scopes_disabled():
+            CartPosition.objects.create(
+                event=self.event, cart_id=self.session_key, item=self.ticket,
+                price=23, expires=now() + timedelta(minutes=10)
+            )
+
+        r = self.client.get('/%s/%s/checkout/questions/' % (self.orga.slug, self.event.slug))
+        assert b'TRACKING SCRIPT' in r.content
+
+        payment_r = self.client.post('/%s/%s/checkout/questions/' % (self.orga.slug, self.event.slug), {
+            'is_business': 'business',
+            'company': 'Foo',
+            'name': 'Bar',
+            'street': 'Baz',
+            'zipcode': '1234',
+            'city': 'Here',
+            'country': 'AT',
+            'email': 'admin@localhost'
+        }, follow=True)
+        assert b'TRACKING SCRIPT' not in payment_r.content
+
```

Comment:
Should we test all possible payment pages/URLs listed under „Zahlungsseiten“ on our PCI-DSS-page to make sure that all of them correctly set the `request.pci_dss_payment_page = true`?

Reply:
PCI rule 11.6.1 will require us to build a "monitoring" for the scripts on all payment pages on pretix.eu anyways, so I think we can skip that.